rfp-proposals/rfp-proposal-sarahos.md

```diff
@@ -0,0 +1,66 @@
+
+# RFP Proposal: `SarahOS`
+
+**Name of Project:** SarahOS
+
+**Link to RFP:** [https://github.com/IPFS-grandhelmsman/sarah-doc](https://github.com/IPFS-grandhelmsman/sarah-doc)
+
+**RFP Category:** `devtools-libraries`
+
+**Proposer:** `IPFS-grandhelmsman`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** "Yes"
+
+# Project Description
+
+Problem: if a developer wants to run and manage multiple ipfs-based projects on multiple servers, he should first solve the operation and maintenance of hardware nodes, including the reading of hardware running status information, alarm, etc, and next step is the projects of batch management.
+
+Solution: under our system, developers only need to add and optimize the projects on the platform we provide. Refer to the Sarah OS product documentation，there are a more detailed description.
```

Comment:
Could you share the link to Sarah OS product documentation? The product links don't seem to include documentation.

Reply:
https://github.com/IPFS-grandhelmsman/sarah-doc